src/Common/src/System/Security/Cryptography/Asn1/asn.xsd

```diff
@@ -159,6 +159,21 @@
     <xs:attribute name="twoDigitYearMax" use="optional" type="xs:unsignedShort" />
   </xs:complexType>
 
+  <xs:complexType name="ObjectIdentifierType">
```

Comment:
Currently all of the complex types are in their UNIVERSAL tag ordering.  So OID (0x06) would go between Integer (0x02) and Enumeration (0x0A).

I shouldn't care, but I can't not.

Reply:
Makes sense, I will move it over.